src/test/java/uk/ac/cam/cl/dtg/segue/api/monitors/MisuseMonitorTest.java

```diff
@@ -77,151 +80,116 @@ public final void misuseMonitorTokenOwnerLookup_checkForMisuse_emailShouldBeSent
-      try {
-        misuseMonitor.notifyEvent(userId, event);
-        if (i >= TokenOwnerLookupMisuseHandler.HARD_THRESHOLD) {
-          fail(String.format("Exception not thrown after %s attempts, over limit of %s!", i,
-              TokenOwnerLookupMisuseHandler.HARD_THRESHOLD));
-        }
-      } catch (SegueResourceMisuseException e) {
-        if (i < TokenOwnerLookupMisuseHandler.HARD_THRESHOLD) {
-          fail(String.format("Exception thrown before %s attempts, under limit of %s!", i,
-              TokenOwnerLookupMisuseHandler.HARD_THRESHOLD));
-        }
-      }
+      assertDoesNotThrow(() -> misuseMonitor.notifyEvent(userId, event),
+          String.format("Exception thrown before %d attempts, under limit of %d!", i,
+              tokenOwnerLookupMisuseHandler.getHardThreshold()));
     }
 
+    assertThrows(SegueResourceMisuseException.class, () -> misuseMonitor.notifyEvent(userId, event),
+        String.format("SegueResourceMisuseException not thrown after %d attempts, over limit of %d!",
+            tokenOwnerLookupMisuseHandler.getHardThreshold(),
+            tokenOwnerLookupMisuseHandler.getHardThreshold()));
+
     verify(this.dummyCommunicator, this.dummyPropertiesLoader);
   }
 
   /**
    * Verifies that the email verification misuse handler is working.
    */
   @Test
-  public final void emailVerificationRequest_checkForMisuse_emailShouldBeSentAndExceptionShouldOccur() {
+  final void emailVerificationRequest_checkForMisuse_emailShouldBeSentAndExceptionShouldOccur() {
 
     String event = EmailVerificationRequestMisuseHandler.class.getSimpleName();
 
     IMisuseMonitor misuseMonitor = new InMemoryMisuseMonitor();
 
-    EmailVerificationRequestMisuseHandler emailVerificationMisuseHandler
-        = new EmailVerificationRequestMisuseHandler();
+    EmailVerificationRequestMisuseHandler emailVerificationMisuseHandler = new EmailVerificationRequestMisuseHandler();
 
     misuseMonitor.registerHandler(event, emailVerificationMisuseHandler);
 
-
     // Create a test user
     RegisteredUser user = new RegisteredUser();
     user.setEmail("test@test.com");
     user.setEmailVerificationStatus(EmailVerificationStatus.NOT_VERIFIED);
 
     // Soft threshold
-    try {
-      //Register the misuse monitor
-      if (misuseMonitor.hasMisused(user.getEmail(),
-          EmailVerificationRequestMisuseHandler.class.getSimpleName())) {
-        throw new SegueResourceMisuseException("Number of requests exceeded. Triggering Error Response");
-      }
-
-      for (int i = 0; i < EmailVerificationRequestMisuseHandler.SOFT_THRESHOLD; i++) {
-        misuseMonitor.notifyEvent(user.getEmail(), event);
-      }
-    } catch (SegueResourceMisuseException e) {
-      fail();
+    for (int i = 0; i < emailVerificationMisuseHandler.getSoftThreshold(); i++) {
+      assertDoesNotThrow(() -> misuseMonitor.notifyEvent(user.getEmail(), event),
+          String.format("Exception should not be thrown after %d attempts",
```

Comment:
Wrong message, as you're printing the threshold, rather than the number of attempts.